bundles/binding/org.openhab.binding.insteonplm/src/main/resources/device_features.xml

```diff
@@ -18,6 +18,15 @@
 	<poll-handler>NoPollHandler</poll-handler>
 </feature>
 
+<feature name="RampOnOff">
+	<message-dispatcher>DefaultDispatcher</message-dispatcher>
+	<message-handler cmd="0x19">RampOnOffSwitchHandler</message-handler>
```

Comment:
Isn't this ignored by the rampOnOffSwitchHandler? I only see 0x34 and 0x35 in the Handler.